### PR TITLE
[In Progress] POS

### PR DESCRIPTION
**Pull request by iros**
Working on POS tagging.

So far added install_dependencies for nltk packages:

`textkit install_dependencies`

Will do the trick.

### CODE REVIEW

**Comment by vlandham** [comment]
would it be possible to call the public facing command `download` or `downloaddata` ? 

The download would make it a bit more explicit that this command is getting something from a remote location. "install" perhaps might be more appropriate when compiling or configuring a piece of software?

I've also been trying to avoid dashes or underscores in the names. Perhaps this is misled - but when they are used in other tools, i always forget which one (dash or underscore) is used.

File: textkit/install_dependencies.py [hunk 1]
```diff
@@ -0,0 +1,50 @@
+import os
+import click
+import nltk
+from textkit.utils import read_tokens, output
+
+
+@click.command('install_dependencies')
+@click.option('--path', 
+              help='Separator between token and count in output.',
+              show_default=True)
```
Comment (vlandham) [inline textkit/install_dependencies.py]: is there going to be a default? I see show_default is true - but I don't see a default.

File: textkit/install_dependencies.py [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+
+def check_packages_exist(extensions):
+	'''
+	Finds missing nltk extensions.
+	'''
+	paths = nltk.data.path # there are usually quite a few, so we check them all.
+	missing = []
+	ext_found = False
+	for ext_tuple in extensions:
+		print("Looking for " + ext_tuple[1])
```
Comment (vlandham) [inline textkit/install_dependencies.py]: you might consider using [click echo](http://click.pocoo.org/5/api/#click.echo) here too. its supposed to help with unicode issues and python 2/3 stuffs

**Comment by vlandham** [comment]
The other thing we could do is just copy over the necessary files into our `data` directory. This has already been done for the stop word lists. If the file size is small, this would simplify the users interaction with textkit and ensure we always had a consistent location for the necessary data files.

File: textkit/install_dependencies.py (hunk 1 above)
Comment (iros) [inline textkit/install_dependencies.py]: This is actually optional. I couldn't find a way to specify this! I saw `required=True` in the docs.... so not sure.

**Comment by iros** [comment]
@vlandham, pushed a commit to rename to "download" and replaced the prints with echo.

As far as putting things in the data dir, I will investigate.

**Comment by iros** [comment]
@vlandham I pushed a fix that added pos tagging. I tried your suggestion of must adding the specific tagger to the data dir and that works. If you're cool with this approach, I will remove the download task.
The only challenge might be version mismatch between our tagger and the nltk version someone might already have on their machine. Not sure if that's actually something to be concerned over.

Let me know what you think.

File: textkit/download.py [hunk 3]
```diff
@@ -0,0 +1,50 @@
+import os
```
Comment (iros) [inline textkit/download.py]: We may not need this file anymore... see my last commit.

**Comment by iros** [comment]
Well, I screwed this up majorly by trying to remove that one file... UGH. Sorry ya'll. I'll move this into another PR later. Otherwise the code is ready for review, so we can do that here if that makes sense @vlandham.

**Comment by vlandham** [comment]
looks great and excited to have it! 

do we still need `textkit/install_dependencies.py` ? it is still in the modified files - but I think you replaced it with `download` right?

**Comment by vlandham** [comment]
Closing this one - as it has been already done in #30

**Pull request closed without merging**